Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.Events.cs

```diff
@@ -0,0 +1,33 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// In App Notification defines a control to show local notification in the app.
+    /// </summary>
+    public partial class InAppNotification
```

Comment:
Could we also have events for Opened/Opening/Closed/Closing like [ContentDialog](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.contentdialog#events_) / [FlyoutBase](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.flyoutbase#events_)?

I would see closed/closing fire whenever it disappears (including dismiss or timeout) vs. the Dismiss event which fires when the user explicitly dismisses the notification.